### PR TITLE
Fix daemonize

### PR DESCRIPTION
**Pull request by jettero**
the old salt.utils.daemonize was itself a deprecated form of salt.utils.process.daemonize; ... so I added those methods back to hubblestack.utils.daemonize

### CODE REVIEW

**Comment by devagupt** [comment]
The changes here looks good. I will merge this PR after verifying that build from it runs fine on a machine.

**Review by devagupt — COMMENTED** [review]
The changes here looks good. I will merge this PR after verifying that build from it runs fine on a machine.

**Comment by devagupt** [comment]
The automation tests are not passing for this build. We need to fix them as well.
Test results can be accessed here:
https://hubble-cicd.corp.adobe.com/job/Hubble_Automation_jobs_stage/job/Hubble_Automation_centos7_stage/40/testReport/

**Review by MoodyMudit — CHANGES_REQUESTED** [review]

File: hubblestack/files/hubblestack_nova/fdg.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def audit(data_list, tags, labels, debug=False, **kwargs):
                 true_for_success = tag_data.get('true_for_success', True)
                 use_status = tag_data.get('use_status', False)
 
-                _, fdg_run = __mods__['fdg.fdg'](fdg_file, starting_chained=starting_chained)
+                _, fdg_run = __mods__['fdg.run'](fdg_file, starting_chained=starting_chained)
```
[the remark this answers is not in the document]
Comment (jettero) [reply hubblestack/files/hubblestack_nova/fdg.py:99]: wait, this is the "old fdg" ? meaning this file should be removed? something might be pretty screwy. Or maybe I just don't understand what you mean.

[the remark this answers is not in the document]
Comment (MoodyMudit) [reply hubblestack/files/hubblestack_nova/fdg.py:99]: As discussed, this is the old fdg, can you please revert back this change.

**Review by MoodyMudit — CHANGES_REQUESTED** [review]

**Review by goravsingal — APPROVED** [review]

**Review by MoodyMudit — APPROVED** [review]

**Review by goravsingal — APPROVED** [review]

**Review by MoodyMudit — APPROVED** [review]

**Review by goravsingal — APPROVED** [review]

**Pull request merged**